addons/hr/models/resource.py

```diff
@@ -1,11 +1,26 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ResourceResource(models.Model):
     _inherit = "resource.resource"
 
     user_id = fields.Many2one(copy=False)
     employee_id = fields.One2many('hr.employee', 'resource_id', check_company=True)
+
+    job_title = fields.Char(related='employee_id.job_title')
+    department_id = fields.Many2one(related='employee_id.department_id')
+    work_email = fields.Char(related='employee_id.work_email')
+    work_phone = fields.Char(related='employee_id.work_phone')
+    show_hr_icon_display = fields.Boolean(related='employee_id.show_hr_icon_display')
+    hr_icon_display = fields.Selection(related='employee_id.hr_icon_display')
+
+    @api.depends('employee_id')
+    def _compute_avatar_128(self):
+        for resource in self:
+            employee = resource.with_context(active_test=False).employee_id
```

Comment:
> is that active_test needed even for a many2one? 😲

It was introduced here: https://github.com/odoo/enterprise/commit/08581f7c8895a0f2774e68d4c837b7290776d0b0

Even if no test were added, I suppose, reading the commit message that it is still helpfull to keep that context. I'm not aware of a change since then about active_test not needed for m2o :man_shrugging:

Reply:
I'm also changing a bit this method, following https://github.com/odoo/enterprise/commit/7bf295cf069db35eb9cff5157c9143da5a8e920d

Reply:
Can you please test? I really don't think it is needed.

Reply:
> Can you please test? I really don't think it is needed.

I've tested in my pr (with the slight changes that I had done in the compute method). It's is needed apparently even for a M2O. Even more, I don't need it only for the avatar but also for other fields of the employee record to correctly display the avatar card of a resource linked to an archive employee. Therefore I've directly added the context in the field definition.

Reply:
> Can you please test? I really don't think it is needed.

Regarding this, I've moved the test `test_availability_hr_infos_resource` from planning to hr and modify the test case to make the hr.employee record inactive.